### PR TITLE
add details to readme

### PR DESCRIPTION
**Pull request by mermatriarchy**
Adds details to the ReadMe & updates links for better documentation & contributing instructions
Addresses issue #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,14 +8,36 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
...
-`git clone https://github.com/mermatriarchy/fuma.git`<br>
+`git clone https://github.com/flatbush-united/fuma.git`<br>
 `npm install`<br>
 `npm run start-local`<br>
 
+## Hosting & Deployment
+
+This project is currently hosted on Heroku under Hailey's account. In the near future, it would be good to get away from using a personal Heroku account & host the site on a more centralized platform (something like GitHub pages or a FUMA group account for hosting services).
+
+Currently, deployments are handled by Hailey via the Heroku cli. We don't have any CI/CD solutions implemented at the moment.
```
Comment (phalaenid) [inline README.md:19]: In the Heroku Config settings, there should be a UI where you can connect deployments from Github (so every change to a certain branch triggers deploys).  That would probably be the lowest friction way to get that setup while we try and figure out a more long term hosting / CI/CD workflow.

Comment (mermatriarchy) [reply README.md:19]: Ohhh, never knew it was this easy! I'll set up this config & update the readme accordingly

Comment (phalaenid) [reply README.md]: Awesome, thank you!!!

**Comment by phalaenid** [comment]
@mermatriarchy thank you for your work on this, I think this looks great and am going to merge knowing we can always iterate.  One thing in the future we might want to add is a Code of Conduct alongside the contributor guidelines (the [Contributor Covenant ](https://www.contributor-covenant.org/)might be a good starting point that we could adapt based on the overall group's visioning language).

**Pull request merged**